### PR TITLE
[#175761632] Add CGN Activation API

### PR DESCRIPTION
**Pull request by AleDore**
#### List of Changes
- Add `startCgnActivation` API implementation
- Add tests

#### Motivation and Context
Provide an IO Backend proxying and implementation of the `startCgnActivation` [API](https://github.com/pagopa/io-functions-cgn/pull/13). See the related Pivotal [story](https://www.pivotaltracker.com/story/show/175761632) for further informations.

#### How Has This Been Tested?
It has been tested by performing unit tests.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by balanza** [comment]
can you fix target branch?

File: src/services/cgnService.ts [hunk 1]
```diff
@@ -69,4 +80,51 @@ export default class CgnService {
...
+              response.value,
+              fromNullable(response.headers.Location).getOrElse(
+                "/api/v1/cgn/activation"
+              ),
+              response.value
+            );
+          case 202:
+            return ResponseSuccessAccepted();
+          case 401:
+            return ResponseErrorUnexpectedAuthProblem();
```
Comment (balanza) [inline src/services/cgnService.ts:116]: What scenario can cause this?

Comment (AleDore) [reply src/services/cgnService.ts:103]: Only a misconfiguration ( missing or wrong API key)

**Comment by AleDore** [comment]
Maybe it's better to merge this [one](https://github.com/pagopa/io-backend/pull/751) first, so we can perform a correct rebase @balanza @BurnedMarshal

**Comment by AleDore** [comment]
Ok, Can we merge now @balanza @BurnedMarshal ?

**Review by balanza — APPROVED** [review]

**Pull request merged**